react-components/src/components/RuleBasedOutputs/utils/getResourceRelationship.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type RelationshipResourceType, type CogniteClient } from '@cognite/sdk/dist/src';
+import { type ExtendedRelationshipWithSourceAndTarget } from '../../../data-providers/types';
+import { getRelationships } from '../../../hooks/network/getRelationships';
+
+export const getResourceRelationship = async (
```

Comment:
Maybe I'm blind, but I don't see where this is used :thinking: Is it necessary?

Reply:
Actually it was exposed for the usage in Fusion. But I think it is better to wrap it in a hook and consume the hook instead. I changed the code here:
https://github.com/cognitedata/reveal/pull/4805/commits/bbe2ff5eb1fa6034338c9b47445419e23f968c85